config/samples/varnishcluster.yaml

```diff
@@ -12,8 +12,6 @@ spec:
     # path to image + tag
     image: ibmcom/varnish:0.27.1
     imagePullPolicy: Always
-    # restart policy used for Varnish containers
-    restartPolicy: Always
```

Comment:
was there anything wrong with that config?
I don't mind commenting it if necessary, but let's have it here so users can see all available fields with comments in this sample.

Reply:
I tried to apply the base VarnishCluster CRD and it returned an error saying this field did not exist.

Reply:
ah, right. We removed it when we moved to statefulsets where `Always` is the only allowed value. Good catch!